src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -182,35 +186,37 @@ export default function TokenTable({
     }
   }, [page, prevPageIndex])
 
-  // reset table to page 1 on chain change
-  useEffect(() => {
-    if (chainId !== prevChainId) {
-      setPage(1)
-    }
-  }, [chainId, prevChainId])
-
-  if (!tokensData) {
-    return <Loader />
-  }
-
   return (
     <Wrapper>
       <ErrorModal />
       <TransactionConfirmationModal />
-      {sortedTokens.length > 0 ? (
+      {tokensData && sortedTokens.length !== 0 ? (
         <AutoColumn>
           <Table ref={tableRef}>
             <TableHeader>
-              <Label>#</Label>
+              <IndexLabel>#</IndexLabel>
               <ClickableText onClick={() => handleSort(SORT_FIELD.NAME)}>
                 <Trans>Name {arrow(SORT_FIELD.NAME)}</Trans>
               </ClickableText>
               <ClickableText disabled={!account} onClick={() => (account ? handleSort(SORT_FIELD.BALANCE) : false)}>
                 <Trans>Balance {arrow(SORT_FIELD.BALANCE)}</Trans>
               </ClickableText>
+              <Label>Value</Label>
               <Label>Buy</Label>
               <Label>Sell</Label>
-              <Label>Approve</Label>
+              <Label>
+                <span>Approve</span>
+                <MouseoverTooltip
+                  text={
+                    <Trans>
+                      Enable token for trading. This only need to be done once. Once it is enabled, you can place orders
```

Comment:
where is this text coming from?

Reply:
From the comment Anxo made https://github.com/cowprotocol/cowswap/pull/677#issuecomment-1178796972